### DISCUSSION TITLE
Early Access release models with other integrations

### DISCUSSION THREAD

**Discussion by hassan-sd** [civitai/civitai#172] [Ideas]
Allow model creators to "whitelist" a content release for certain Civitai users who meet whatever criteria they set out.

**Use case**
* As a model creator, I can release a newly trained model to a group of whitelisted users who can test and provide feedback on a test version of my content.
_I can differentiate between whitelisted users vs non whitelisted users_
_I can add users to a whitelist based on their civitai username or other automatic random criteria such as X activity, Y comments, Z reviews/ratings etc_
* As a model creator I can whitelist **patron** users as part of [another feature request](https://github.com/civitai/civitai/discussions/171) to gain early access to my content and provide feedback via civitai when whitelisted
* The content that is released in early access can then be pushed to full release access to all users within the same content/model page, just another "version" that is not whitelisted
![image](https://user-images.githubusercontent.com/119671806/214038403-02d5926f-2ce1-4a40-8f9b-83602c1cffe6.png)

![image](https://user-images.githubusercontent.com/119671806/214039077-89452283-3c85-4e4c-8c04-7364ca8ef3ff.png)

**Comment by JustMaier** [comment]
Thanks for suggesting this! We'll be adding and improving creator tools over the next few months and so it's important to get input like this.

We're currently focused on a gallery aspect of the site, which we hope will bring you as a creator an even bigger audience. The next focus area is things that improve your ability to monetize your work. That will include things like this. I'll be sure to follow up when we get to that stage.
